packages/api/prisma/schema.prisma

```diff
@@ -0,0 +1,38 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+  output   = "./client"
+}
```

Comment:
Second, we need to output the client to a directory within the api package rather than in the node modules directory which is the default. This allows multiple packages to have their own prisma client tailored to their own database. Instead of importing `@prisma/client` we just have to import from `./prisma/client` within the api package